internal/venice-client-common/src/main/java/com/linkedin/venice/stats/AbstractVeniceStats.java

```diff
@@ -92,7 +92,8 @@ protected Sensor registerSensor(
               }
             }
           } else {
-            String metricName = sensorFullName + "." + stat.getClass().getSimpleName();
+            String statName = stat.getClass().getSimpleName();
+            String metricName = sensorFullName + "." + (statName.equals("LongAdderRateGauge") ? "Rate" : statName);
```

Comment:
This is now handled as part of another change. You'll see it if you rebase...

Reply:
yup, I know :) I need to rebase